docs/Queries/Line Continuations.md

```diff
@@ -0,0 +1,28 @@
+---
+publish: true
+---
+
+# Line Continuations
+> [!released]
+> Introduced in Tasks X.Y.Z.
+
+Long queries can be organized across multiple lines using a trailing backslash:
+
+    ```tasks
+    (priority is highest) OR \
+        (priority is lowest)
+    ```
+
+This can be helpful for long [[Combining Filters|combined filters]],  [[Custom
+Filters|custom filters]], and other queries that may be difficult to read on one line.
+
+Line continuations are treated as whitespace, and any extra indentation or whitespace around the backslash is compressed into a single space.
+
+In the rare case that a trailing backslash is needed for a query, the trailing backslash may be escaped by adding a space at the end of the line, or by adding an extra backslash and a blank line:
```

Comment:
I also preferred the previous behavior, where two trailing slashes got reduced to a single slash. However, I couldn't find a way to encode that behavior in a regex without using a lookbehind. The current behavior is to treat a trailing double slash as "normal backslash"+"line continuation".

I'm fine with omitting the blank line trick from the docs and just supporting the space trick. I personally prefer not to have trailing whitespace, since it sometimes gets automatically stripped by editors.

Reply:
> I also preferred the previous behavior, where two trailing slashes got reduced to a single slash. However, I couldn't find a way to encode that behavior in a regex without using a lookbehind. The current behavior is to treat a trailing double slash as "normal backslash"+"line continuation".

Ok thank you, that's a really helpful comment, to distinguish between the two behaviours.

> I'm fine with omitting the blank line trick from the docs and just supporting the space trick. I personally prefer not to have trailing whitespace, since it sometimes gets automatically stripped by editors.

OK, I'll think about that.

Reply:
> I also preferred the previous behavior, where two trailing slashes got reduced to a single slash.

Resolving. I'll pick this up after merging the PR.

Reply:
> I also preferred the previous behavior, where two trailing slashes got reduced to a single slash.

- Done in 82c1abb69f69b9c383cd057e1f289566d55005a9
- Docs updated in 0b0a355908b70821a59216f227a42e0a4d730e4b

(I forgot to create a branch so it's on main, without a PR...)